src/graphql-server/schema.graphql

```diff
@@ -106,6 +106,7 @@ type HposStatus {
 
 type HposAuth {
   isAuthed: Boolean
+  hostSettings: HposSettings
```

Comment:
can we just call this field `hposSettings`
in less there's a specific reason not to, it's good to name your variables the same name as their type.

Reply:
The type name was made to match/reflect the endpoint name, but I agree that the gql type should match the var name.  Update done. :+1: